apidoc2swagger/apidoc2swagger.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ *
+ * @file
+ * @version 1.0
+ * @copyright 2017 CN-Consult GmbH
+ * @author Pascal Stiemer <pascal.stiemer@cn-consult.eu>
+ */
```

Comment:
Since this file does not only contain a class, but is a script that can be called you should also document the parameters it uses.